app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java

```diff
@@ -120,7 +118,10 @@ public void onResult(AppStateManager.StateResult stateResult) {
 
                     @Override
                     public void success(final Directory directory, final retrofit.client.Response response) {
-                        String[] orgas = taggedOrganizers.split("\\|");
+                        if (orgas != null) {
+                            return;
+                        }
+                        orgas = taggedOrganizers.split("\\|");
```

Comment:
Yep, no problem. This was a bit of a mess when you started. I'm trying to submit a PR tonight that should help clean it up some.